modules/interactive-list.js

```diff
@@ -0,0 +1,30 @@
+import { tasks, addElemToPage } from './toDos.js';
+
+// ======Checked Function=======
+window.updateStatus = (selectedTask) => {
+  // console.log(selectedTask);
+  const taskName = selectedTask.nextElementSibling;
+  // console.log(taskName);
+  if (selectedTask.checked) {
+    taskName.classList.add('checked');
+    // console.log(tasks[selectedTask.id].completed);
+    tasks[selectedTask.id - 1].completed = true;
+    // console.log(tasks[selectedTask.id].completed = true);
+  } else {
+    taskName.classList.remove('checked');
+    tasks[selectedTask.id - 1].completed = false;
+  }
+  localStorage.setItem('task-list', JSON.stringify(tasks));
+};
+
+// ======Clear Completed Function=======
+const clearCompletedBtn = document.querySelector('.clear');
+
+clearCompletedBtn.addEventListener('click', () => {
+  tasks = tasks.filter((task) => !task.completed);
+  tasks.forEach((e, i) => {
+    e.index = i + 1;
+  });
+  localStorage.setItem('task-list', JSON.stringify(tasks));
+  addElemToPage();
+});
```

Comment:
- [ ] Nice work you did here. however, the function does not clear the tasks completed. kindly fix this bug to complement the fantastic job you've done so far 👏 

## View :
![image](https://user-images.githubusercontent.com/50990925/220907999-06af6ee8-a937-4272-86c8-346ba40a7031.png)

Reply:
Hi @Shedrack-Sunday, Thanks for your efforts in reviewing my code 😄 .